ddtrace/contrib/asgi/middleware.py

```diff
@@ -77,6 +77,11 @@ class TraceMiddleware:
         tracer: Custom tracer. Defaults to the global tracer.
     """
 
+    @staticmethod
+    def handle_exception_span(exc, span):
```

Comment:
Maybe people would like to use the request that generated the error.

It's the default behavior in fastAPI for a [custom error handler ](https://fastapi.tiangolo.com/tutorial/handling-errors/#install-custom-exception-handlers)
 
something like :
```python
def handle_exception_span(request,exc, span): 
```

Reply:
In this generic ASGI middleware, we do not have access to a request/response interface as you would find in something like with a Starlette middleware's dispatch method. 

We are working now on support for a Starlette middleware, where we could certainly provide library users a way to accomplish this.

Reply:
Okay it make sense , thank you for your answer.